extensions/amp-subscriptions/0.1/test/test-local-subscriptions.js

```diff
@@ -45,4 +52,43 @@ describes.realWin('local-subscriptions', {amp: true}, env => {
     expect(initializeStub.getCall(0).args[1].credentials)
         .to.be.equals('include');
   });
+
+  describe('initActions_', () => {
+    it('should build actions and urlbuilder on constructor', () => {
+      expect(localSubscriptionPlatform.urlBuilder_).to.be
+          .instanceof(UrlBuilder);
+      expect(localSubscriptionPlatform.actions_).to.be
+          .instanceof(Actions);
+      expect(JSON.stringify(localSubscriptionPlatform.actions_.actionsConfig_))
+          .to.be.equal(JSON.stringify(actionMap));
+    });
+  });
+
+  describe('validateActionMap', () => {
+    let actionMap;
+    beforeEach(() => {
+      actionMap = {
+        'subscribe': 'https://lipsum.com/subscribe',
+        'login': 'https://lipsum.com/login',
```

Comment:
Test with one more non-standard action as well.

Reply:
any example of what is needed to be tested?

Reply:
E.g. add also `'other': 'https://lipsum.com/other'` and have a test to ensure it's there and executable - i.e. not dropped or errors out.